lib/publiccloud/azure.pm

```diff
@@ -82,7 +82,7 @@ sub find_img {
     eval {
         $image = decode_azure_json($json)->{name};
     };
-    record_info('INFO', "Cannot find image $name. Need to upload it.\n$@") if ($@);
+    record_info('IMG NOT-FOUND', "Cannot find image $name. Need to upload it.\n$@") if ($@);
```

Comment:
NO NEED TO YELL!!1!

```suggestion
    record_info('Image not present', "Cannot find image $name. Need to upload it.\n$@") if ($@);
```

Reply:
It's not yelling. We use uppercase letters in those boxes.

Reply:
Ok